csrankings-z.csv

```diff
@@ -173,7 +173,7 @@ Zhihui Lu 0002,Fudan University,http://www.cs.fudan.edu.cn/?page_id=3647,NOSCHOL
 Zhihui Wang,Fudan University,http://www.cs.fudan.edu.cn/?page_id=1958###,NOSCHOLARPAGE
 Zhijia Zhao 0001,Univ. of California - Riverside,http://www.cs.ucr.edu/~zhijia,klBvLwYAAAAJ
 Zhijian Wu,Wuhan University,http://cs.whu.edu.cn/teacherinfo.aspx?id=220,NOSCHOLARPAGE
-Zhijiao Xiao,Shenzhen University, https://www.scholarmate.com/P/ZhijiaoXiao
+Zhijiao Xiao,Shenzhen University,https://www.scholarmate.com/P/ZhijiaoXiao

```

Comment:
Missing Google Scholar ID.

Reply:
No Google Scholar ID. Thanks.

Reply:
Per the instructions:
>  Update affiliations, home pages, and Google Scholar entries by modifying csrankings-[a-z].csv. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

Reply:
(You can literally see this in the lines just above your change.)